src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -1294,6 +1294,13 @@ public virtual string Labels
         {
             get
```

Comment:
This accessor is unit tested [HERE](https://github.com/newrelic/newrelic-dotnet-agent/blob/e925389eaff3790381467749c71250c586baa194/tests/Agent/UnitTests/Core.UnitTest/Configuration/DefaultConfigurationTests.cs#L1299-L1309), and should be extended to test the the new app local config override functionality you are adding.

I would suggest matrixing in a parameter to test that the ConfigurationManager based override works as expected. Examples on how to mock ConfigurationManger can be found in the same file.